### ISSUE TITLE
The password does not meet the length, complexity, or history requirement of the domain.

### ISSUE THREAD

**Issue by GoKEV** [mgmt-sa-tiger-team/skylight#53]
```
TASK [windows-ad-controller : Add windows domain admin] ******************************************************************************************************************************************************
fatal: [windc]: FAILED! => {"changed": false, "msg": "The password does not meet the length, complexity, or history requirement of the domain.", "password_updated": false}

```

Even when using the copy-paste examples in README.md file, deployment fails with this error.  I tried a couple of passwords that *should* meet general complexity; with no success.

Perhaps the complexity requirements could be dialed back to something that would potentially be "dangerous" in other environments?  Some might argue that a complex password is actually unnecessary in this environment.

**Comment by cigamit** [comment]
Some might argue that since these are usually spun up in public clouds and are accessible to everyone on the internet, that the password requirements should be increased even more.

We would have to modify group policy to change the default policy, and I don't think it's necessary.

**Comment by cigamit** [comment]
Fixed via 51febbe63074e8839fb227279aa47ea475085c7c
